shell/common/gin_converters/frame_converter.cc

```diff
@@ -18,4 +20,51 @@ v8::Local<v8::Value> Converter<content::RenderFrameHost*>::ToV8(
   return electron::api::WebFrameMain::From(isolate, val).ToV8();
 }
 
+// static
+v8::Local<v8::Value>
+Converter<gin_helper::AccessorValue<content::RenderFrameHost*>>::ToV8(
+    v8::Isolate* isolate,
+    gin_helper::AccessorValue<content::RenderFrameHost*> val) {
+  auto* rfh = val.Value;
+  if (!rfh)
+    return v8::Null(isolate);
+
+  const int process_id = rfh->GetProcess()->GetID();
+  const int routing_id = rfh->GetRoutingID();
+
+  auto rfh_templ = v8::ObjectTemplate::New(isolate);
```

Comment:
Is it OK to create a new ObjectTemplate every time this function is called? I know that FunctionTemplates are supposed to only be created once per isolate, since they're never GC'd. Is the same true for ObjectTemplates..?

e.g. in event_emitter.cc, we hold a global `v8::Persistent<v8::ObjectTemplate>` and only set it once.

Reply:
I wasn't able to find much documentation saying to do otherwise, but I did find sample code within v8 using a Persistent handle. https://source.chromium.org/chromium/chromium/src/+/main:v8/test/cctest/test-api.cc;l=14037-14043;drc=8ab75a56a24f34d4f582261c99939ffa1446a3b7

I ended up doing that for the RFH template.